src/main/credentialServer/README.md

```diff
@@ -1,75 +0,0 @@
-# Credential Helper Server protocol
```

Comment:
We should still have a readme (though it doesn't need to describe the protocol anymore).

Reply:
Added a smaller one back, that basically references the upstream repository to read more about the protocol (which is defined there only in loose prose).